products/green_fast_track/seeds/variables.csv

```diff
@@ -12,3 +12,9 @@ arterial_highways,Arterial Highways,Noise,16,"75 foot buffer"
 elevated_railways,Elevated Railways,Noise,17,"1,500 foot buffer"
 airport,Airport,Noise,18,"Intersects 65db contour"
 e_designations_hazmat,(E) Designations - HazMat,HazMat,HZ1,"HazMat Code = 1"
+nyc_parks_properties,NYC Parks properties,Shadow/Open Space,23,"75 foot buffer. Specific list of allowed typecategory values"
+pops,POPS,Shadow/Open Space,23,"75 foot buffer around BBL lot"
+waterfront_access_wpaa,Waterfront Public Access Areas,Shadow/Open Space,23,"75 foot buffer"
+waterfront_access_pow,Publicly Owned Waterfront,Shadow/Open Space,23,"75 foot buffer"
+nys_parks_properties,NYS Parks Property,Shadow/Open Space,23,"75 foot buffer"
+us_parks_properties,Federal Parks property,Shadow/Open Space,23,"75 foot buffer"
```

Comment:
Don't have to do it now, but probably worth adding a check to ensure that all these variables are represented in "all flags" table, and that "all flags" doesn't have any that aren't present here. I need to wrap up my PR so I can include that